projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -77,6 +77,11 @@ export interface FeatureTogglesInterface {
    */
   storeFrontLibCardParagraphTruncated?: boolean;
 
+  /**
+   * In `ProductCarouselComponent` it uses the batch API to load products.
+   */
```

Comment:
```suggestion
 /**
   * When enabled, the batch API is used `ProductCarouselComponent` to load products. It increases the component's performance.
   *
   * _NOTE_: When flag is enabled, custom OCC config for the `productSearch` endpoint has to be adjusted to have an object representation:
   * ```js
   * backend: {
   *    occ: {
   *      endpoints: {
   *         productSearch: {
   *           default: '...',
   *           carousel: '...',
   *           carouselMinimal: '...',
   *         },
   *       },
   *     },
   *   }
   * ```
   */
```

Reply:
Done